### DISCUSSION TITLE
taiko gauge: Need to check/wait until UI is completely load before executing every step?

### DISCUSSION THREAD

**Discussion by qa22swsdsr** [getgauge/taiko#2525] [General]
taiko gauge: Need to check/wait until UI is completely loaded before executing every step?

**Comment by NivedhaSenthil** [comment]
Taiko tries to wait for page load to done implicitly and the definition of UI is completely loaded may vary from site to site.. you could try waiting for specific events please refer https://docs.taiko.dev/frequently_asked_questions/#how-can-i-customize-taiko's-wait-events%3F or wait for custom events or elements using `waitFor` api.
